clients/admin-ui/src/features/privacy-requests/events-and-logs/ActivityTimeline.tsx

```diff
@@ -35,70 +29,86 @@ const hasUnresolvedError = (entries: ExecutionLog[]) => {
   });
 
   // Check if any of the latest entries for the collections have an error status.
-  return Object.values(groupedByCollection).some(
-    (entry) => entry.status === ExecutionLogStatus.ERROR,
-  );
+  return Object.values(groupedByCollection).some((entry) => {
+    // For entries with a collection_name, check for later complete status
+    if (entry.collection_name) {
+      const latestComplete = entries.find(
+        (e) =>
+          e.status.toLowerCase() === "complete" &&
+          !e.collection_name && // completion entries have null collection_name
```

Comment:
It seems flaky to rely on a null `collection_name` to check for complete entries. What do you think?

Reply:
This is for the `Data traversal` timeline entry. Any errors will be associated with a collection but a successful traversal will show up with a `complete` status and no collection